frontend/views/Information/Information.js

```diff
@@ -1,78 +1,255 @@
 import React from "react";
 import Main from "../../components/Main";
 import Fullscreen from "../../components/Layout/Fullscreen";
+import { ComposableMap, Geographies, Geography } from "react-simple-maps";
 
 import InformationStyles from "./Information.module.scss";
 import Success from "components/Success";
+import useMediaQuery from "hooks/useMediaQuery.hook";
+
+const allCountries = [
+  'Albania',
+  'Aland',
+  'Andorra',
+  'Austria',
+  'BrusselsCapitalRegion',
+  'FlemishRegion',
+  'Bulgaria',
+  'BosniaandHerzegovina',
+  'RepublicSrpska',
+  'Belarus',
+  'WalloonRegion',
+  'Switzerland',
+  'CzechRepublic',
+  'Germany',
+  'Denmark',
+  'England',
+  'Estonia',
+  'Spain',
+  'Finland',
+  'FaroeIslands',
+  'France',
+  'Guernsey',
+  'Gibraltar',
+  'Greece',
+  'Croatia',
+  'Hungary',
+  'IsleofMan',
+  'Ireland',
+  'Iceland',
+  'Italy',
+  'Jersey',
+  'Kosovo',
+  'Liechtenstein',
+  'Lithuania',
+  'Luxembourg',
+  'Latvia',
+  'Monaco',
+  'Moldova',
+  'Macedonia',
+  'Malta',
+  'Montenegro',
+  'NorthernIreland',
+  'JanMayen',
+  'Netherlands',
+  'Norway',
+  'Svalbard',
+  'Poland',
+  'Portugal',
+  'Romania',
+  'Russia',
+  'Scotland',
+  'SanMarino',
+  'Serbia',
+  'Vojvodina',
+  'Slovakia',
+  'Slovenia',
+  'Sweden',
+  'Ukraine',
+  'Vatican',
+  'Wales'
+];
+
+// TODO: this needs a second look after we've got list of countries that AI works with
```

Comment:
is ukraine not working ? its hot topic right now ;]

Reply:
no AI for ukraine.
no AI for these too but nobody knows